### PR TITLE
update service description in env & update cache setting

### PR DESCRIPTION
**Pull request by fansi404**
This change is [Reviewable](https://reviewable.io/reviews/koderover/zadig-doc/282)

### CODE REVIEW

File: dev/04.项目/03.environment.md [hunk 1]
```diff
@@ -72,6 +72,10 @@ permalink: /dev/project/env/
 - `更新环境`：当环境对应的服务编排有变更或环境里面的服务 YAML 有变更时，环境中会自动出现更新环境按钮。。
```
Comment (flyer103) [inline dev/04.项目/03.environment.md:72]: There's an extra period at then end.

File: dev/04.项目/03.environment.md [hunk 2]
```diff
@@ -172,7 +180,93 @@ permalink: /dev/project/env/
...
+- `选择服务`：要被托管的服务
+
+### 查看环境
+
+点击不同的环境 Tab 即可查看该环境及环境中被托管服务的详情。
+
+![查看托管环境](./_images/list_host_env.png)
+
+#### 基本信息
+包括 K8s 集群，K8s 命名空间，更新时间，环境状态等信息，其中环境状态有以下几种情况：
```
Comment (flyer103) [inline dev/04.项目/03.environment.md:204]: `包括 K8s 集群，K8s 命名空间，更新时间，环境状态等信息` -> `包括 K8s 集群、K8s 命名空间、更新时间、环境状态等信息`

File: dev/04.项目/03.environment.md [hunk 3]
```diff
@@ -172,7 +180,93 @@ permalink: /dev/project/env/
...
+- `创建中`：正在创建。
+- `正在运行`：环境中所有服务都是正常的 Running 状态。
+- `运行不稳定`：环境中一部分服务处于 Unstable 状态，可能的情况有两种：
+	- 容器服务出现了问题.
```
Comment (flyer103) [inline dev/04.项目/03.environment.md:208]: Remove all of `.`/`。` or use the same format.

File: dev/12.系统设置/11.cluster_manage.md [hunk 4]
```diff
@@ -48,10 +48,12 @@ permalink: /dev/pages/cluster_manage/
 ![修改集群缓存资源配置](./_images/cluster_update_1.png)
 
 说明：
-- `选择存储介质`：支持对象存储和集群存储（PV 资源）。
-- `选择存储资源`：对象存储资源或者集群存储 PV 资源。
-  - 如果使用`对象存储`介质，则使用系统默认对象存储作为存储资源，可在[对象存储管理](/dev/settings/object-storage/)中设置默认对象存储。
-  - 如果使用`集群存储`介质，则可选择集群中已有的存储资源或通过指定 PVC 按需创建新的存储资源。
+- `选择存储介质`：支持对象存储和集群存储（PVC 资源）。
+- `选择存储资源`：如果使用`对象存储`介质，则使用系统默认对象存储作为存储资源，可在[对象存储管理](/dev/settings/object-storage/)中设置默认对象存储。如果使用`集群存储`介质，则可选择集群中已有的存储资源或通过指定 PVC 按需创建新的存储资源。
```
Comment (flyer103) [inline dev/12.系统设置/11.cluster_manage.md:52]: PVC -> StorageClass

File: v1.10.0/12.系统设置/11.cluster_manage.md [hunk 5]
```diff
@@ -48,10 +48,12 @@ permalink: /v1.10.0/pages/cluster_manage/
 ![修改集群缓存资源配置](./_images/cluster_update_1.png)
 
 说明：
-- `选择存储介质`：支持对象存储和集群存储（PV 资源）。
-- `选择存储资源`：对象存储资源或者集群存储 PV 资源。
-  - 如果使用`对象存储`介质，则使用系统默认对象存储作为存储资源，可在[对象存储管理](/v1.10.0/settings/object-storage/)中设置默认对象存储。
-  - 如果使用`集群存储`介质，则可选择集群中已有的存储资源或通过指定 PVC 按需创建新的存储资源。
+- `选择存储介质`：支持对象存储和集群存储（PVC 资源）。
+- `选择存储资源`：如果使用`对象存储`介质，则使用系统默认对象存储作为存储资源，可在[对象存储管理](/v1.10.0/settings/object-storage/)中设置默认对象存储。如果使用`集群存储`介质，则可选择集群中已有的存储资源或通过指定 PVC 按需创建新的存储资源。
```
Comment (flyer103) [inline v1.10.0/12.系统设置/11.cluster_manage.md:52]: PVC -> StorageClass

**Review by flyer103 — APPROVED** [review]

**Review by lilianzhu — APPROVED** [review]

**Pull request merged**